mslib/mscolab/file_manager.py

```diff
@@ -83,11 +84,11 @@ def create_operation(self, path, description, user, last_used=None, content=None
                     self.import_permissions(import_op.id, operation_id, user.id)
             data = fs.open_fs(self.data_dir)
             data.makedir(operation.path)
-            operation_file = data.open(fs.path.combine(operation.path, 'main.ftml'), 'w')
-            if content is not None:
-                operation_file.write(content)
-            else:
-                operation_file.write(mscolab_settings.STUB_CODE)
+            with data.open(fs.path.combine(operation.path, 'main.ftml'), 'w') as operation_file:
+                if content is not None and verify_waypoint_data(content):
+                    operation_file.write(content)
+                else:
+                    operation_file.write(mscolab_settings.STUB_CODE)
```

Comment:
So, this would write STUB_CODE if the content supplied to create_operation does not verify, creating an operation with different content then supplied. I think create_operation should instead behave just like save_file, i.e. error out on invalid content and not creating the operation.

Reply:
this is a different problem and should discussed/done afterwards. This may have impact to server configuration defaults for operations, which we also have for flightpathes. 

I've just seen that we not care on closing the file and solved that too.

Reply:
It is new behavior that is introduced in this PR, so I think it should be discussed here.

Previously, the STUB_CODE was used when no content was specified. Now, the STUB_CODE is used when either no content is specified, or the specified content did not validate. The latter is unintuitive behavior to me, and I think create_operation should just error out in this case.

Reply:
Oh,  then I remove it I didn't want to do that